docs/examples/academy/29. Working with Aggregated Cargo Movements Data.ipynb

```diff
@@ -95,41 +96,32 @@
       "text/plain": [
-       "<Figure size 576x432 with 1 Axes>"
+       "<Figure size 800x600 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "weekly_quantity = cms.groupby('loading_week').sum()\n",
+    "weekly_quantity = cms.groupby('loading_week').sum(numeric_only=True)\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.